cli/definitions/cli.ts

```diff
@@ -83,6 +84,11 @@ export interface IDeploymentAddCommand extends ICommand {
     deploymentName: string;
 }
 
+export interface IDeploymentClearCommand extends ICommand {
```

Comment:
In general the commands should indicate that deployment history is being cleared, not the deployment itself (e.g.:  IDeploymentHistoryClearCommand)